tests/chainer_tests/functions_tests/connection_tests/test_n_step_lstm.py

```diff
@@ -103,28 +103,15 @@ def check_forward(
         testing.assert_allclose(cy.data, e_cy, rtol=1e-4, atol=1e-4)
 
     def test_forward_cpu(self):
-        self.check_forward(self.hx, self.cx, self.xs, self.ws, self.bs, False)
-
-    def test_forward_cpu_volatile(self):
-        self.check_forward(self.hx, self.cx, self.xs, self.ws, self.bs, True)
+        self.check_forward(self.hx, self.cx, self.xs, self.ws, self.bs)
```

Comment:
I removed some test cases from links test. I think each link does not have responsible about volatile flag now because volatile flag is removed. And links can not change their behavior based on volatile flag. So, i don't think we need to test such case. Instead we may need to strengthen tests for Function and Variable.